sdk/python/feast/cli_utils.py

```diff
@@ -195,7 +198,121 @@ def handle_not_verbose_permissions_command(
             [t.__name__ for t in p.types],  # type: ignore[union-attr, attr-defined]
             p.with_subclasses,
             p.name_pattern,
-            p.actions,
-            roles,
+            [a.value.upper() for a in p.actions],
+            sorted(roles),
         ]
     )
+
+
+def fetch_all_feast_objects(store: FeatureStore) -> list[FeastObject]:
+    objects: list[FeastObject] = []
+    objects.extend(store.list_entities())
+    objects.extend(store.list_all_feature_views())
+    objects.extend(store.list_batch_feature_views())
+    objects.extend(store.list_feature_services())
+    objects.extend(store.list_data_sources())
+    objects.extend(store.list_data_sources())
```

Comment:
Duplicate

Reply:
You are missing OnDemandFeatureView and StreamFeatureView

Reply:
This was the updated recently by Francisco to return all types of FV:
```py
    def list_all_feature_views(
        self, allow_cache: bool = False, tags: Optional[dict[str, str]] = None
    ) -> List[Union[FeatureView, StreamFeatureView, OnDemandFeatureView]]:
```

Reply:
Great